### PR TITLE
chore(workflows): define setup-helm action patch version

### PR DESCRIPTION
**Pull request by Teko012**
## what
Bump helm to the latest version

## why
The pipeline is failing, because `3.10` is not a valid version number. It should be `3.10.0` or `3.10.3`, which is the latest.

## references
[Failed run](https://github.com/runatlantis/helm-charts/actions/runs/3859256677/jobs/6578609819)
[Helm 3.10.0](https://github.com/helm/helm/releases/tag/v3.10.0)
[Helm 3.10.3](https://github.com/helm/helm/releases/tag/v3.10.3)

### CODE REVIEW

File: .github/workflows/release.yaml
Comment (nitrocode) [inline .github/workflows/release.yaml:25]: Can we omit the version so we're always using the stable version?

https://github.com/Azure/setup-helm#example

Comment (Teko012) [reply .github/workflows/release.yaml:25]: I guess there is a pro and con to it as there is a note there about the API rate limit, which if the runner hits, it defaults back to their hard coded version, which might not be the latest. If we control the version, it's guaranteed that it won't run it with an older version then the previous build because of it.

Comment (Teko012) [reply .github/workflows/release.yaml:25]: @nitrocode Should I update it then?

Comment (nitrocode) [reply .github/workflows/release.yaml:25]: I defer to the other @runatlantis/maintainers here

Comment (Teko012) [inline .github/workflows/release.yaml:25]:
```suggestion
          version: latest
          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment (Teko012) [reply .github/workflows/release.yaml:25]: I added the other option as a suggestion.

**Comment by Teko012** [comment]
@nitrocode What should we do here?

**Review by nitrocode — APPROVED** [review]

**Comment by nitrocode** [comment]
Thank you @Teko012 !

**Pull request merged**
